templates/android/library/src/main/java/io/appwrite/Client.kt.twig

```diff
@@ -85,7 +85,10 @@ class Client @JvmOverloads constructor(
             "content-type" to "application/json",
             "origin" to "{{ spec.title | caseLower }}-android://${context.packageName}",
             "user-agent" to "${context.packageName}/${appVersion}, ${System.getProperty("http.agent")}",
-            "x-sdk-version" to "{{spec.title | caseDash}}:{{ language.name | caseLower }}:${BuildConfig.SDK_VERSION}"{% if spec.global.defaultHeaders | length > 0 %},{% endif %}
+            "x-sdk-name" to "{{ sdk.name }}",
+            "x-sdk-platform" to "{{ sdk.platform }}",
+            "x-sdk-language" to "{{ language.name | caseLower }}",
+            "x-sdk-version" to "{{ sdk.version }}"{% if spec.global.defaultHeaders | length > 0 %},{% endif %}
             
```

Comment:
I don't think empty line is required here

Reply:
It was already present so removing it will most likely change the formatting in the SDKs. Do we want that ?